Frontend/src/CustomScripts/convertData.mjs

```diff
@@ -110,4 +110,17 @@ const newFileOutput = {
 // console.log(output)
 writeFileSync("./src/Components/Graph/graph-data.json", JSON.stringify(newFileOutput, null, 2))
 
-// export {}
+// write this JSON object
+const output = JSON.stringify(json, null, 2);
+// console.log(output);
+writeFileSync("./src/Components/Graph/graph-data.json", output);
+
```

Comment:
```suggestion
```
We don't want to overwrite the content that is being written to `graph-data.json` above.